libr/core/cmd_anal.c

```diff
@@ -729,6 +729,16 @@ static const char *help_msg_ax[] = {
 	NULL
 };
 
+static const char *help_msg_axt[]= {
+	"Usage:", "ax[?gq*]", "find data/code references",
+	"axtj", " [addr]", "find data/code references to this address and print in json format",
+	"axtg", " [addr]", "display commands to generate graphs according to the xrefs",
+	"axtq", " [addr]", "find and list the data/code references in quiet mode",
+	"axt*", " [addr]", "set flags on the corresponding xrefs",
```

Comment:
axt* like other * commands doesn't actually set anything it only prints r2 commands.